agents/src/job.ts

```diff
@@ -190,8 +190,8 @@ export class JobContext {
   onParticipantConnected(p: RemoteParticipant) {
     for (const callback of this.#participantEntrypoints) {
       if (
-        p.identity in this.#participantTasks &&
-        this.#participantTasks[p.identity].callback == callback
+        this.#participantTasks[p.identity] &&
+        this.#participantTasks[p.identity]!.callback == callback
```

Comment:
```suggestion
        this.#participantTasks[p.identity]?.callback == callback
```
Nit: Could simplify by coalescing since if its nullish, it would not be equal